src/app/routes/podcast/index.js

```diff
@@ -0,0 +1,12 @@
+import { RadioPodcastPage } from '#pages';
+import { podcastPath, podcastBrandPath } from '#utils/regex';
+import getInitialData from './getInitialData';
+import { MEDIA_PAGE } from '#utils/pageTypes';
+
+export default {
+  path: [podcastPath, podcastBrandPath],
+  exact: true,
+  component: RadioPodcastPage,
+  getInitialData,
+  pageType: MEDIA_PAGE,
```

Comment:
I haven't worked on the Chartbeat stuff before but guessing the addition of a new page type here is something that will need changing in a later PR? 

Not related to this PR but - any idea why we segmented out `MEDIA_ASSET_PAGE`, `ARTICLE_PAGE`, `STORY_PAGE` etc. but then use `MEDIA_PAGE` across OD Tv, live radio & OD radio?

Reply:
Yes, we need to update Chartbeat before the page goes live. There is a ticket in the milestone which covers the work.

I don't know why it was done that way. It would be good to disentangle them as it leads to quite a lot of extra logic in Chartbeat and ATI to differentiate the page types.